counsel-gtags.el

```diff
@@ -245,8 +248,8 @@ Always update if value of this variable is nil."
          (with-temp-buffer
            (let* ((options (cl-case counsel-gtags-path-style
                              (absolute "-Poa")
-                             (root "-Poc")
-                             (relative ""))))
+                             (root "-Poa")
```

Comment:
Why did you change this option ? `-Poa` shows absolute path, not path from root directory.